src/backend/commands/explain_gp.c

```diff
@@ -941,14 +941,20 @@ cdbexplain_collectStatsFromNode(PlanState *planstate, CdbExplain_SendStatCtx *ct
  */
 typedef struct CdbExplain_DepStatAcc
 {
-	/* vmax, vsum, vcnt, segmax */
+	/* vmax, vmin, vsum, vcnt, segmax, segmin */
 	CdbExplain_Agg agg;
 	/* max's received StatHdr */
 	CdbExplain_StatHdr *rshmax;
 	/* max's received inst in StatHdr */
 	CdbExplain_StatInst *rsimax;
 	/* max's inst in NodeSummary */
 	CdbExplain_StatInst *nsimax;
+	/* min's received StatHdr */
+	CdbExplain_StatHdr *rshmin;
+	/* min's received inst in StatHdr */
+	CdbExplain_StatInst *rsimin;
+	/* min's inst in NodeSummary */
+	CdbExplain_StatInst *nsimin;
```

Comment:
The new added 3 fields `rshmin, rsimin,nsimin` seems not used.

Reply:
Removed